search/search.php

```diff
@@ -20,4 +20,6 @@
 
 require_once __DIR__ . '/includes/classes/class-search.php';
 
-do_action( 'vip_search_loaded' );
+if ( \Automattic\VIP\Search\Search::are_es_constants_defined() ) {
```

Comment:
Now that we're having to repeat this "loading" logic in multiple places, it may make sense to move the logic from 000-vip-init.php into here? We could leave the conditional loading of search/search.php, but bring the JP filtering and instance code into here so we have a single conditional?

Reply:
Nice that is good idea, I couldn't figure out how to do it:)